company/models.py

```diff
@@ -74,7 +74,7 @@ class Company(models.Model):
     title                = models.CharField(max_length=50)
     likes_count          = models.IntegerField(default=0)
     contents             = models.CharField(max_length=1000)
-    image_url            = models.CharField(max_length=200)
+    image_url            = models.CharField(max_length=500)
```

Comment:
```suggestion
    image_url            = models.URLField(max_length=2000)
```
길이는 더 여유있게 주셔도 좋습니다. URLField는 사실 form을 사용할때 validation의 의미가 있는 필드지만, 장고에서 지원해주는 필드로 유의미 하게 위코드에서는 컨벤션으로서 적용하는것을 권장드립니다.  변경해주세요.

Reply:
넵 수정했습니다.